draft-kcs-tvr-requirements.xml

```diff
@@ -272,12 +272,33 @@ In cases where the properties of a link have time-variable values the model <bcp
       </section>
       <section anchor="routing">
         <name>Routing</name>
-        <t>Existing routing techniques are not typically designed to handle potential connectivity, i.e., nodes and links
-           scheduled to appear in the future. Therefore, the TVR Scenarios and use cases referenced and discussed in this
-           document will be compromised of scheduled resources that are expected to appear in the future.</t>
+        <t>Traditional network routing techniques typically use link bandwidth and delay for path calculation, and do not consider time-based factors. TVR should be
+	   capable of improving network performance and reliability in environments where node livelyness and link availability is a time-based consideration, with various factors, 
+           including power availability, interface line of sight or expected demand. 
         <t>However, even if some adjacency failures are predictable, others are not, including link failures and node outages.
            Therefore, any new technique or routing protocol extension for TVR enviroments must be capable of handling planned and unexpected
            resource losses.</t>
+	<t>Time-Variant Routing (TVR) introduces a scenario of calculating a path, or sub-path within a network, taking into account the timing
+	   of message transmission or receipt as an integral part of the overall route computation.</t>
+	<t>Furthermore, Synchronization of network time across TVR-capable nodes is critical in TVR networks.</t> 
```

Comment:
Question here. Does actually TVR take into account the timing of the message transmission (or receipt)? In any case, if the answer is yes, we mention that TVR introduce such scenario. What is the use case supporting it? Maybe better to refer here to the use case document for clarification on which use case this statement is based

Reply:
It's a good point, and related to the scope of what "TVR" really means. Just attaching a schedule to entities in a data model allows expressing time-varying topology and existing routing algorithms can use that data as-is but _will not_ include any time-of-transmission delay in their routing calculations. Other, TVR-aware routing algorithms _can_ take advantage of the schedule and expected delays in their route computing.

I tried to address this in the [Schedule Visibility](https://danielkinguk.github.io/tvr-requirements/5-intrinsic-extrinsic-schedule/draft-kcs-tvr-requirements.html#name-schedule-visibility) section in the `5-intrinsic-extrinsic-schedule` branch.

Reply:
How about adding a "Under Discussion" (need to find a better name) appendix at the end of the document for unanswered questions/requirements. Basically a catchall for anything that needs further discussion.